airflow/ui/src/pages/Variables/Variables.tsx

```diff
@@ -0,0 +1,260 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import {
+  Box,
+  Button,
+  createListCollection,
+  HStack,
+  Input,
+} from "@chakra-ui/react";
+import type { ColumnDef } from "@tanstack/react-table";
+import { useState, useEffect } from "react";
+import { FiUpload, FiFilter } from "react-icons/fi";
+import { IoAddCircleOutline } from "react-icons/io5";
+
+import { useVariableServiceGetVariables } from "openapi/queries";
+import type { VariableResponse } from "openapi/requests/types.gen";
+import { DataTable } from "src/components/DataTable";
+import { useTableURLState } from "src/components/DataTable/useTableUrlState";
+import { ErrorAlert } from "src/components/ErrorAlert";
+import { CloseButton, Select } from "src/components/ui";
+
+const variablesColumn = (): Array<ColumnDef<VariableResponse>> => [
```

Comment:
This can just be an array instead of a function that returns an array.
```suggestion
const columns: Array<ColumnDef<VariableResponse>> = [
```